### PR TITLE
Added hometown information

### LINKED ISSUE #76: Create hometown file
#Italy

### PR DESCRIPTION
**Pull request by umanganiello**
This resolves #76

### CODE REVIEW

File: umanganiello-atripalda.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Atripalda, AV
+
```
Comment (umanganiello) [inline umanganiello-atripalda.md]: @vbontempi Please add a line here quatzy

**Comment by vbontempi** [comment]
@umanganiello Che faccio? Ammacco?

**Comment by umanganiello** [comment]
@vbontempi smarmella

**Comment by vbontempi** [comment]
it works by fear

**Pull request merged**